### ISSUE TITLE
Support --create-pr flag

### ISSUE THREAD

**Issue by FezVrasta** [flow-typed/flow-typed#4042]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
Right now, to send a PR with new types, I need to manually create a PR, which involves forking the repository, create a branch, make the changes, commit, push, and send the PR.

**If the current behavior is a bug, please provide the steps to reproduce.**

Following an approach similar to `patch-package --create-issue` flag, we could have a `create--pr` flag that will automatically create a PR with the type definitions that the user created or modified locally.

**What is the expected behavior?**
```
$ flow-typed create-pr <package>@<version>
Please insert a valid GitHub access token with repo:write permissions:
*************
A new Pull Request with your contribution has been created and it's available at https://github.com/flow-typed/flow-typed/pulls/xxxx
```

**If this is a feature request, what is motivation or use case for changing the behavior?**
The current approach can limit the amount of contributions to this project.

**Local Environment Information**
If this issue is specific to the CLI, provide the results of running `npx envinfo` from the command line in a GitHub gist.

**Comment by pascalduez** [comment]
Hi @FezVrasta,

could be nice.
How would that work with tests files? :sweat_smile:

**Comment by FezVrasta** [comment]
One can add them to the same local directory and the script would match them (prepend test_ to the file name)?

**Comment by Brianzchen** [comment]
Didn't know these things existed, is there any project that does something similar to this?

Updating existing defs is simple, but creating a new one need some finesse to make all the types with the correct naming scheme. Even if we don't do what you're suggesting, some cli command that automates initial file creation would be great.

**Comment by AndrewSouthpaw** [comment]
Yeah never heard of `patch-package` and it'd be cool to see something like that in the CLI.

**Comment by Brianzchen** [comment]
@FezVrasta Wondering do you have ideas of any additional features you'd like here or is the `./create-def.sh` enough which has been good enough for my usage.

**Comment by FezVrasta** [comment]
@Brianzchen I didn't have time to test it yet, I'm sorry.
